pkg/modelzooserver/image_build.go

```diff
@@ -16,13 +16,18 @@ package modelzooserver
 import (
 	"bytes"
 	"fmt"
+	"io"
 	"io/ioutil"
 	"net/http"
 	"os"
 	"os/exec"
 	"strings"
+
+	"sqlflow.org/sqlflow/pkg/randstring"
 )
 
+const kanikoImage = "registry.cn-hangzhou.aliyuncs.com/sql-machine-learning/kaniko-executor"
```

Comment:
Maybe we can use the official Dockerhub registry to be the default value, and specify another registry by an environment variable.

Reply:
Changed to use env, yet kept use aliyun image as default.